<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(219, 236, 255)
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widget_2" native="true"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 700 italic 23pt &quot;Georgia&quot;;</string>
        </property>
        <property name="text">
         <string>Your profile!</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QWidget" name="widget_5" native="true"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>First Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_6" native="true"/>
      </item>
      <item>
       <widget class="QLineEdit" name="firstname">
        <property name="styleSheet">
         <string notr="true">background: white
</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_7" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_8" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QWidget" name="widget_9" native="true"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Last Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_10" native="true"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lastname">
        <property name="styleSheet">
         <string notr="true">background: white
</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_11" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_12" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QWidget" name="widget_13" native="true"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Date of Birth</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_14" native="true"/>
      </item>
      <item>
       <widget class="QLineEdit" name="DOB">
        <property name="styleSheet">
         <string notr="true">background: white
</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_15" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_16" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QWidget" name="widget_17" native="true"/>
      </item>
      <item>
       <widget class="QPushButton" name="bmain1_2">
        <property name="styleSheet">
         <string notr="true">background:rgb(248, 251, 255)</string>
        </property>
        <property name="text">
         <string>Update Profile</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_18" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_19" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="bmain1">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(190, 213, 255)</string>
        </property>
        <property name="text">
         <string>Back to Main</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_20" native="true"/>
      </item>
      <item>
       <widget class="QWidget" name="widget_21" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
